magento/adobe-stock-integration
# #1609: [Refactoring] DateTime handling and formatting

### Need to verify (*)
- **Created** and **Modified** values are saved and displayed without any issues when viewing image details

### Description (*)
- Refactored datetime handling and formatting by adding/using Magento Framework DateTime functionality 

### Fixed Issues (if relevant)
1. Fixes magento/adobe-stock-integration#1609: [Refactoring] DateTime handling and formatting

### Manual testing scenarios (*)
1. Go to **Content - Media Gallery**
2. Search for any image from the **Search Adobe Stock** button
2. Select the added image from Media Gallery and click "three dots" in the bottom right
3. Select **View Details**
4. Check **Created** and **Modified** values under **Details**
